vehicle/vehicle_info_util/scripts/min_turning_radius_calculator.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import math
+
+from ament_index_python.packages import get_package_share_directory
+import yaml
+
+
+def main(yaml_path):
+    with open(yaml_path) as f:
+        config = yaml.safe_load(f)
+
+    wheel_base = config["/**"]["ros__parameters"]["wheel_base"]
+    max_steer_angle = config["/**"]["ros__parameters"]["max_steer_angle"]
+
+    print("yaml path is {}".format(yaml_path))
+    print("Minimum turning radius is {} [m]".format(wheel_base / math.sin(max_steer_angle)))
```

Comment:
It'd be better to show the definition of the radius since the formula depends on how you define the radius. If it is for the rear wheel, the equation should be 
```
radius = wheel_base / tan(steer)
```

If it is for front steer, it will be 
```
radius = wheel_base / sin(steer)
```

Maybe it is nice to print both, like
```
Minimum turning radius is 4.40 [m] for rear, 4.45[m] for front
```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1855/commits/50746d6800418c7f751001dac4966049851ba1dc
DONE

Reply:
cool!